tests/inverted-index-testSpec.js

```
@@ -1,7 +1,201 @@
-import myApp from '../src/inverted-index.js';
+import fs from 'fs';
+import myApp from '../src/inverted-index';
 
-describe('Sum()', () => {    
-  it('counts one word', () => {
-    expect(myApp.sum(1, 2)).toEqual(3);
+const jsonOfFile = new myApp('book.json');
+const emptyJson = new myApp('empty.json');
+const invalidJson = new myApp('invalid.json');
+const malformedJson = new myApp('malformed.json');
+const malformed1Json = new myApp('malformed1.json');
+const validJson = new myApp('bookone.json');
+const multipleFiles = new myApp(['book.json', 'bookone.json']);
+
+describe('Inverted index class', () => {
+  describe('Read book data', () => {
+    it("Should return 'malformed json' for fileContent = ''", () => {
+      expect(invalidJson.getJson()).toEqual('malformed json');
+    });
+
+    it("Should return '[{}]' for fileContent = '[{}]'", () => {
```

It should return a proper error response and not just an empty object literal.